src/wasm/asm-wasm-builder.cc

```diff
@@ -26,36 +26,35 @@ namespace wasm {
 
 class AsmWasmBuilderImpl : public AstVisitor {
  public:
```

Comment:
Since this class is local to the file, I wouldn't bother with public/private separation. That way, you can inline the contents of the static method Run() into the caller AsmWasmBuilder::Run().